server.js

```diff
@@ -50,6 +50,23 @@ app.use("/test/jasmine",
     )
 );
 app.get("/test/katex-spec.js", serveBrowserified("test/*[Ss]pec.js"));
+// Serve the test html file but with the doctype removed to trigger quirks
+// mode.
+app.get("/test/screenshotter/test-quirks.html", function(req, res, next) {
```

Comment:
Indeed, that's something left over from how @xymostech approached this. Should be removed I think.